common/js/src/popup-window/in-popup-main-script.js

```diff
@@ -155,7 +157,7 @@ GSC.InPopupMainScript.setWindowHeightToFitContent = function() {
     chrome.app.window.current().innerBounds.height = wholeContentHeight;
   } else if (GSC.Packaging.MODE === GSC.Packaging.Mode.EXTENSION) {
     chrome.windows.update(
-        chrome.windows.WINDOW_ID_CURRENT, {'height': wholeContentHeight});
+        chrome.windows.WINDOW_ID_CURRENT, {'height': wholeContentHeight + titleBarHeight});
```

Comment:
nit: Please add a comment explaining why we need to do that - so the code reader can understand why we do it here and why we don't do it in the "if (GSC.Packaging.MODE === GSC.Packaging.Mode.APP)" case above.